Remotion/Web/Development.WebTesting.UnitTests/Configuration/IConfiguration/WebTestSettingsDtoTest.cs

```diff
@@ -0,0 +1,200 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+#if !NETFRAMEWORK
+using System;
+using System.IO;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.Configuration;
+using Remotion.Web.Development.WebTesting.Configuration.IConfiguration;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium;
+
+namespace Remotion.Web.Development.WebTesting.UnitTests.Configuration.IConfiguration
+{
+  [TestFixture]
+  public class WebTestSettingsDtoTest
+  {
+    private const string c_testConfig = @"{
```

Comment:
one config is a field, the other is inline. we use both only once. I would prefer a symmetric approach.